NewSpring/ApollosProject
# Adds How to Pray Button Functionality

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR allows a person to tap on the "How to Pray" button and see the correct article.

### How do I test this PR?
Go to any of the prayer feeds and click on the "How to Pray" button and see the article. Whoo!

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [n/a] Closes [DEV-XXX](#url)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
